kcsd/validation/VisibilityMap.py

```diff
@@ -562,8 +735,18 @@ def plot_error_map(self, point_error, ele_pos):
                         ele_lims=ELE_LIMS, true_csd_xlims=TRUE_CSD_XLIMS,
                         est_xres=0.01, est_yres=0.01)
     rms, point_error = a.calculate_error_map(CSD_PROFILE,
-                                             Rs=np.arange(0.05, 0.5, 0.05),
-                                             lambdas=np.array(0), n=2)
+                                              Rs=np.arange(0.05, 0.5, 0.05),
+                                              lambdas=np.array(0), n=2)
+
+    print('Checking 2D MoI')
+    ELE_LIMS = [0.118, 0.882]
+    CSD_PROFILE = CSD.gauss_2d_large
+    a = VisibilityMap2DMoI(total_ele=9, h=50., sigma=1., n_src_init=1000,
+                            ele_lims=ELE_LIMS, true_csd_xlims=TRUE_CSD_XLIMS,
+                            est_xres=0.01, est_yres=0.01)
```

Comment:
I see a pattern here.  But it is inconsistent with horizontal alignment of call arguments used in the project.